nextflow.config

```
@@ -55,9 +55,15 @@ params {
     databases = null
 
     // FASTQ preprocessing
-    shortread_clipmerge           = false
-    shortread_excludeunmerged     = true
-    longread_clip                 = false
+    shortread_clipmerge                     = false
+    shortread_clipmerge_tool                = 'fastp'
+    shortread_clipmerge_skiptrim            = false
+    shortread_clipmerge_mergepairs          = false
+    shortread_clipmerge_excludeunmerged     = true
+    shortread_clipmerge_adapter1            = null
+    shortread_clipmerge_adapter2            = null
+    shortread_clipmerge_minlength           = 15
```

As mentioned above, how about we change this so that parameters can be accessed like `params.shortread.clipmerge.tool`.

```suggestion
    shortread = [clipmerge: [tool: 'fastp', ...]]
```